tox.ini

```diff
@@ -8,10 +8,8 @@ skip_missing_interpreters=True
 
 [testenv]
 deps =
-    -r{toxinidir}/tests/requirements/requirements_base.txt
```

Comment:
I added this because I thought that the isort and flake8 deps needed it, obviously not so no issues removing this.